bootstrap.js

```diff
@@ -4,42 +4,49 @@
 
 Components.utils.import("resource://gre/modules/Services.jsm");
 
-function startup(data, reason)
-{
-  Services.prefs.setBoolPref("dom.vr.enabled", true);
+const PREFS = {
+  // Enables WebVR. Not needed for Nightly, but we need to keep for other versions.
+  "dom.vr.enabled": true,
```

Comment:
As per gfx.vr.mirror-textures, the reason not to set to "True" would be performance.  Additionally, we don't want content to expect that mirroring is assumed as the upcoming API changes expect mirroring to be done by the rendering engine (three.js, aframe, etc)